fontc/src/lib.rs

```diff
@@ -1259,6 +1263,37 @@ mod tests {
             .collect::<Vec<_>>()
     }
 
+    #[test]
+    fn compile_generates_notdef() {
+        let temp_dir = tempdir().unwrap();
+        let build_dir = temp_dir.path();
+        let result = compile(Args::for_test(build_dir, "glyphs2/WghtVar.glyphs"));
+
+        assert!(!result
+            .fe_context
+            .preliminary_glyph_order
+            .get()
+            .contains(&GlyphName::NOTDEF));
+        assert_eq!(
+            Some(0),
+            result
+                .fe_context
+                .glyph_order
+                .get()
+                .glyph_id(&GlyphName::NOTDEF)
+        );
+
+        let font_file = build_dir.join("font.ttf");
+        assert!(font_file.exists());
+        let buf = fs::read(font_file).unwrap();
+        let font = FontRef::new(&buf).unwrap();
+
+        assert_eq!(
+            GlyphId::new(0),
+            font.cmap().unwrap().map_codepoint(0u32).unwrap()
```

Comment:
again, this is wrong, NULL character (0x0000) should not be mapped to ".notdef" glyph, i.e. the first glyph (gid=0) in the font. They are different things.